src/ResourceManagement/Compute/ComputeManagement/Generated/VirtualMachineScaleSetsOperationsExtensions.cs

```diff
@@ -767,12 +767,12 @@ public static void StartInstances(this IVirtualMachineScaleSetsOperations operat
             /// <param name='vmScaleSetName'>
             /// The name of the virtual machine scale set.
             /// </param>
-            /// <param name='vmInstanceIDs'>
-            /// The list of virtual machine scale set instance IDs.
+            /// <param name='instanceIds'>
+            /// Gets or sets the virtual machine scale set instance ids.
             /// </param>
-            public static void BeginStartInstances(this IVirtualMachineScaleSetsOperations operations, string resourceGroupName, string vmScaleSetName, VirtualMachineScaleSetVMInstanceIDs vmInstanceIDs)
+            public static void BeginStart(this IVirtualMachineScaleSetsOperations operations, string resourceGroupName, string vmScaleSetName, IList<string> instanceIds = default(IList<string>))
```

Comment:
It seems by default it's an empty list, so the request body is most likely ```{}```. Not sure whether this works or not... will give it a try in test re-recording.